charts/helm-dashboard/templates/deployment.yaml

```diff
@@ -40,6 +40,12 @@ spec:
             value: /opt/dashboard/helm/config
           - name: HELM_DATA_HOME
             value: /opt/dashboard/helm/data
+          - name: DEBUG
+            {{- if .Values.debug }}
+            value: 1
+            {{- else }}
+            value: 0
```

Comment:
AFAIK any non-empty value would enable debugging. So intead of `0` it should be empty string. Also I'd enclosse `1` into quotes to make it YAML string